streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -3926,38 +3925,6 @@ t2p1, new PartitionInfo(t2p1.topic(), t2p1.partition(), null, new Node[0], new N
         );
     }
 
-    @ParameterizedTest
```

Comment:
Why have you removed these tests?

Reply:
This PR removes `public AsyncKafkaConsumer<byte[], byte[]> consumer()` which is not needed any longer, but the test rely on it.